### ISSUE TITLE
Different text vertical alignment on Android/Windows/macOs with default font

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-jb#89]
Android | Windows | macOs
![Android](https://user-images.githubusercontent.com/5963351/98843168-a394e480-245b-11eb-8ae6-79b287b14fc2.png) ![Windows](https://user-images.githubusercontent.com/5963351/98843175-a4c61180-245b-11eb-8de5-aea8da639d29.png) ![macOs](https://user-images.githubusercontent.com/5963351/98843173-a42d7b00-245b-11eb-962a-d280cf626446.png)

```
Column {
    Button(onClick = {}) {
        Text("Text")
    }
    Box(Modifier.background(Color.Red)) {
        Text("Text")
    }
    Box(Modifier.background(Color.Green).height(30.dp)) {
        Text("Text", Modifier.align(Alignment.CenterStart))
    }
}
```

**Comment by igordmn** [comment]
Experiments with:

```
Text("Tet", Modifier.background(Color.Red), fontSize = 600.sp)
```

Segoe UI (default font on windows):
![image](https://user-images.githubusercontent.com/5963351/107052451-bad73a00-67de-11eb-8c85-0192955cd54c.png)

Helvetica Neue (default font on macOs):
![image](https://user-images.githubusercontent.com/5963351/107052469-bf9bee00-67de-11eb-980a-b2862bd80ed3.png)


I ran this code:
```
    val typeface = Typeface.makeFromFile("Helvetica.ttf")
//    val typeface = Typeface.makeFromFile("HelveticaNeue.ttf")
    val font = Font(typeface, 1200f)
    println(typeface.familyName)
    println(font.metrics.ascent)
    println(font.metrics.descent)
```

(HelveticaNeue.ttf and Helvetica.ttf are from /System/Library/Fonts, extracted from ttc, Helvetica is the default font of Skia and Chrome)

And it prints on Windows:
```
Helvetica Neue
-1142.4
255.6

Helvetica
-1140.2344
270.1172
```
On macOs:
```
Helvetica Neue
-1142.395
255.59692

Helvetica
-924.02344
275.97656
```

So it seems on macOs we have wrong ascent with the same font file.

P.S. Chrome hasn't any issues with vertical alignment with Helvetica.

**Comment by igordmn** [comment]
Fixed in 1.0.0-beta5

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6244">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
